go.mod

```diff
@@ -1,18 +1,20 @@
 module github.com/openshift/cert-manager-operator
 
-go 1.22.0
+go 1.23
```

Comment:
we need to use go 1.22 for now, as this feature is targetted for cert-manager 1.15 which will be based on current go version only.